### DISCUSSION TITLE
Bug for the Explore page

### DISCUSSION THREAD

**Discussion by Ai686Leo** [community/community#148024] [Mobile] [answered]
I haven't seen the explore page about releases in over a month. Even though the developers released a new version and I chose to follow it in the filters, I haven't seen it since the bug that prioritized the old info on the explore page was fixed

**Comment by lemorage** [comment]
I am wondering the same thing...

**Comment by aletschfirn** [comment]
Yeah... been a long time since it happened https://github.com/orgs/community/discussions/146744

**Comment by josiah999-dot** [comment]
#####- [ ] **@**![image](https://github.com/user-attachments/assets/8aa84a5d-56f3-40d2-9506-6590af71a769)

**Reply by Ai686Leo** [reply]
？？？

**Comment by Mkolibr** [comment]
Mobi

**Comment by Ai686Leo** [comment, marked as answer]
The problem has been solved
